### DISCUSSION TITLE
当我把我的有道x3s从词典笔os降级后它变得无法激活

### DISCUSSION THREAD

**Discussion by JimmyGuo12** [PenUniverse/PenMods#219] [你问我答]
mac和sn_都没改过，但就激活不了，求帮助

**Comment by Eganchiyu** [comment]
直接找有道客服，人家两下从线上操作就可以激活了，我当时不小心改了mac，但是不记得原来的了，急坏了，后来找客服激活，他们相当于把云端数据改成现在的数据了，也就是说，我现在的词典笔能激活，但mac是08:0B:11:22:33:44（我真的笑）
